sdk/core/Azure.Core/src/Pipeline/HttpPipeline.cs

```diff
@@ -34,42 +33,46 @@ public HttpPipeline(HttpPipelineTransport transport, HttpPipelinePolicy[]? polic
         public Request CreateRequest()
             => _transport.CreateRequest();
 
+        public HttpPipelineMessage CreateMessage()
```

Comment:
Should this take a cancellation token? Otherwise I can create a message and send it without every having to pass CT, nor having the ability to pass CT.

Reply:
You can pass it by setting the message property. I would rather put cancellation token onto SendAsync if you insist we have it as parameter somewhere, at least this way we have symmetry with SendRequestAsync

Reply:
If it's in sendasync, I am fine.